### PR TITLE
Cleaned AFK checking

### PR DESCRIPTION
**Pull request by gerdoe-jr**
Using `bool` instead of `int` for `m_SendNAfkMsg`
Deleted `m_pAfkMsg` from `CPlayer` declaration
Cleaned last activity check code

Please test this PR, because I don't know what can go wrong here :P

## Checklist

- [x] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by def-** [comment]
> Please test this PR, because I don't know what can go wrong here :P

You can set the afk time lower and test it yourself locally

**Comment by gerdoe-jr** [comment]
Seems I forgot to add another commit to it, uh. Will push it somewhen xD

File: src/game/server/player.cpp [hunk 1]
```diff
@@ -702,7 +699,7 @@ void CPlayer::TryRespawn()
 		m_pCharacter->SetSolo(true);
 }
 
-bool CPlayer::AfkTimer(int NewTargetX, int NewTargetY)
+bool CPlayer::AfkTimer(CNetObj_PlayerInput *NewTarget)
```
Comment (Robyt3) [inline src/game/server/player.cpp:702]: `pNewTarget`

File: src/game/server/player.h [hunk 2]
```diff
@@ -182,19 +177,16 @@ class CPlayer
 
 	bool m_Moderating;
 
-	bool AfkTimer(int new_target_x, int new_target_y); //returns true if kicked
+	bool AfkTimer(CNetObj_PlayerInput *NewTarget); //returns true if kicked
```
Comment (Robyt3) [inline src/game/server/player.h:180]: `pNewTarget`

File: src/game/server/player.cpp [hunk 3]
```diff
@@ -718,36 +715,36 @@ bool CPlayer::AfkTimer(int NewTargetX, int NewTargetY)
...
-		m_Sent2ndAfkWarning = 0;
+		// we shouldn't update m_pLastTarget here
+		m_Sent1stAfkWarning = false; // afk timer's 1st warning after 50% of sv_max_afk_time
+		m_Sent2ndAfkWarning = false;
 	}
 	else
 	{
 		if(!m_Paused)
 		{
+			char AfkMsg[160];
```
Comment (Robyt3) [inline src/game/server/player.cpp:729]: `aAfkMsg`

**Comment by def-** [comment]
@gerdoe-jr ping, there are unresolved comments

**Comment by gerdoe-jr** [comment]
Tested ingame, works as it should work.
screenshot_2021-08-30_18-04-05

**Pull request merged**